gnomad_qc/v3/load_data/compute_coverage.py

```diff
@@ -12,11 +12,14 @@ def main(args):
     hl.init(default_reference='GRCh38')
     logger = logging.getLogger("gnomad_qc.v3.load_data.compute_coverage")
 
+    logger.warning("Last time this was run (July 2020), this script required high-mem machines.")
```

Comment:
this is a helpful comment, thanks for adding it! it might also be helpful to add the cluster configuration as a comment here or note it somewhere else